source/includes/swift/_objects.md

```diff
@@ -0,0 +1,240 @@
+### Objects or Files
+
+Create and manage your objects/files on a container.
+
+<!-------------------- LIST OBJECTS -------------------->
+
+#### List objects/files
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/swift/example/objects?bucket_name=private"
+
+# The above command returns JSON structured like this:
+```
+
+```json
+{
+  "data": [
+    {
+      "id": "private/myfile_7g.bin-dlo",
+      "name": "myfile_7g.bin-dlo",
+      "displayName": "myfile_7g.bin-dlo",
+      "path": "",
+      "size": 7073741824,
+      "isNode": false,
+      "bucketName": "private",
+      "displaySize": {
+        "value": "6.6",
+        "unitKey": "units.filesizes.GB"
+      },
+      "contentType": "application/octet-stream",
+      "lastModified": "Wed Mar 11 16:51:20 EDT 2020"
+    },
+    {
+      "id": "private/exampleFolder",
+      "name": "exampleFolder",
+      "displayName": "exampleFolder",
+      "path": "",
+      "size": 0,
+      "isNode": true,
+      "bucketName": "private",
+      "contentType": "application/directory",
+      "lastModified": "Tue Mar 10 17:55:28 EDT 2020"
+    },
+    {
+      "id": "private/exampleFolder/sampleFile.txt",
+      "name": "sampleFile.txt",
+      "displayName": "sampleFile.txt",
+      "path": "",
+      "size": 27,
+      "isNode": false,
+      "bucketName": "private",
+      "displaySize": {
+        "value": "27",
+        "unitKey": "units.filesizes.B"
+      },
+      "contentType": "application/octet-stream",
+      "lastModified": "Wed Mar 11 16:51:20 EDT 2020"
+    }
+  ],
+  "metadata": {
+    "recordCount": 3
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/objects?bucket_name=:bucket_name</code>
+
+Retrieve a list of all objects in a given [environment](#administration-environments) in the specified container
+
+Mandatory | &nbsp;
+------- | -----------
+`bucket_name` <br/>*string* | The name of the bucket from which the object must be fetched. 
```

Comment:
Again some more inconsistency here and in the attributes with which sentences end in a period or not

Reply:
Will fix

Reply:
f47db080986d4eed0065f2b048c24c5faeafb7be